inference-engine/src/inference_engine/ie_core.cpp

```diff
@@ -308,17 +265,30 @@ class Core::Impl : public ICore {
 
     std::unordered_set<std::string> opsetNames;
     std::vector<IExtensionPtr> extensions;
-    CacheManager::Ptr cacheManager;
+    std::shared_ptr<ICacheManager> cacheManager;
 
     std::map<std::string, PluginDescriptor> pluginRegistry;
     mutable std::mutex pluginsMutex;  // to lock parallel access to pluginRegistry and plugins
 
+    ExecutableNetwork LoadNetworkImpl(const std::string& modelPath, const std::string& deviceName,
+        const std::map<std::string, std::string> & config,
+        const RemoteContext::Ptr& context) {
+        THROW_IE_EXCEPTION_WITH_STATUS(NOT_IMPLEMENTED) << "mnosov: TBD";
+    }
+
     ExecutableNetwork LoadNetworkImpl(const CNNNetwork& network, const std::string& deviceName,
-                                      std::map<std::string, std::string> _config,
+                                      const std::map<std::string, std::string> & config,
                                       const RemoteContext::Ptr & context) {
-        OV_ITT_SCOPED_TASK(itt::domains::IE_LT, "Core::Impl::LoadNetwork");
+  //      NetworkCompilationContext context(network, compileConfig);
+        return DoLoadNetworkImpl(network, deviceName, config, context);
+    }
+
+    ExecutableNetwork DoLoadNetworkImpl(const CNNNetwork & network, const std::string & deviceName,
```

Comment:
why do we need extra impl function like `DoLoadNetworkImpl`? `LoadNetworkImpl` is already targeted to be such helper function.

Reply:
This is temporary code, will be reworked when I'll implement 'import without read'